front-end/src/app/app.component.ts

```diff
@@ -344,42 +351,70 @@ export class AppComponent extends FullScreen implements OnInit, OnDestroy {
    *
    * @param jsonData with name, camera, buttons, events, hints
    */
-  private processSetUp(jsonData) {
+  private processSetup(jsonData) {
     this.nameOfRoom = jsonData.name;
 
-    const cameraData = jsonData.cameras;
+    this.setupCameras(jsonData.cameras);
+    console.log(jsonData);
+    this.setupButtons(jsonData.buttons);
+    this.resetFrontEndStatus();
```

Comment:
why is this in there - leave this in setupButtons because it has to do with buttons

Reply:
moved resetFrontEndStatus to setupButtons